README.MD

```diff
@@ -79,6 +99,21 @@ strictly part of DAFT but rather DAFT includes tools and libraries to interact
 with the device so own test can be written or existing test suites can be
 integrated.
 
+![](readme_pictures/daft_run.png)
+
+_Running daft._
+
+1. Start daft by executing the following command in the PC Host: `daft DUT PATH/TO/IMG.img`. Replace DUT with the desired DUT e.g. `minnowboard` or `joule` and `PATH/TO/IMG.img` with the image you want to flash. You can also run the command directly in BBB but it then it won't lock the DUTs. Running daft in BBB is done with the command `aft DUT PATH/TO/IMG.img`.In case you want to only flash or test or boot the DUT refer to [DAFT commandline interface](#32-daft-commandline-interface) to add correct options for DAFT.
+2. DAFT then checks if the device is blacklisted in the lockfiles. If not, then it locks it so other simultaneous DAFT runs won't use the same DUT.
```

Comment:
DAFT checks if there is any free device of that type so not blacklisted or locked. Or if you call DAFT with specific device like `joule1` then it will only check that specific lockfile if its locked or blacklisted.